save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/CheckBoxGrid.kt

```diff
@@ -56,4 +71,22 @@ fun checkBoxGrid(options: List<String>) = fc<CheckBoxGridProps> { props ->
                 }
             }
     }
+    useEffect(emptyList<dynamic>()) {
+        js("var jQuery = require(\"jquery\")")
```

Comment:
why not unite it with the next line?

Reply:
I decided to put different expressions into different calls, in order not to make it even more of a mess.